features/admin.extensions.v1/configs/models/sms-provider.ts

```diff
@@ -19,7 +19,7 @@
 import { ReactNode } from "react";
 
 interface SmsProviderConfig {
-    renderAlternativeSmsProviderOptions: () => ReactNode
+    renderAlternativeSmsProviderOptions: (options: any) => ReactNode
```

Comment:
Is it possible to avoid using `any` here?

Reply:
Fixed with [c250bad](https://github.com/wso2/identity-apps/pull/6080/commits/c250badf35df570629e049a5fdbe36eaa67b3b58)